### PR TITLE
Require Node 4+

### PR DESCRIPTION
**Pull request by sapegin**
As discussed in https://github.com/cssinjs/jss/issues/338

You’ll need to publish a new jss version first I think.

### CODE REVIEW

**Comment by sapegin** [comment]
Some here ;-)

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```bash
+npm i
+npm test
+```
+
+Or manually
+
+```bash
+npm run build:tests
+open index.html
```
Comment (kof) [inline contributing.md]: its tests.html

Comment (sapegin) [inline contributing.md]: True! Fixed.

**Comment by kof** [comment]
merged, thanks

**Comment by sapegin** [comment]
Cool, could you please publish new version some time soon?

**Comment by kof** [comment]
I am planing a bigger major release this or next week, do you need it sooner?

**Comment by sapegin** [comment]
No, it’s fine. I’m going to make a major release of Stylegudist (with JSS) next week or a bit later.

**Pull request merged**